package.json

```diff
@@ -1,29 +1,32 @@
 {
   "name": "junknet",
   "version": "0.1.0",
-  "description": "Zero-config distributed parallel processing",
+  "description": "Zero-config distributed parallel processing.",
   "license": "Apache-2.0",
   "files": [
     "dist",
-    "docs"
+    "!**/*.tsbuildinfo"
```

Comment:
We're not using the `--incremental` flag, right? What else would produce `.tsbuildinfo`?

Reply:
`composite: true` implies `incremental: true`